creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -687,6 +687,14 @@ const handleSyncFromPrimary = async ({
  *    with progressively increasing clock values until secondaries have completely synced up.
  *    Secondaries have no knowledge of the current data state on primary, they simply replicate
  *    what they receive in each export.
+ * @param {Object} param
```

Comment:
I had to add these so typescript stopped complaining - ideally this file will get typed soon but out of scope for this PR